spec/requests/tricky_operations_spec.rb

```diff
@@ -168,4 +168,30 @@
       it { is_expected.to be_forbidden }
     end
   end
+
+  describe 'PATCH /articles/:id (nullifying to-one relationship)' do
+    let(:article) { articles(:article_with_author) }
+    let(:json) do
+      <<-EOS.strip_heredoc
+      {
+        "data": {
+          "id": "#{article.external_id}",
+          "type": "articles",
+          "relationships": { "author": null }
+        }
+      }
+      EOS
+    end
+    let(:policy_scope) { Article.all }
+    let(:user_policy_scope) { User.all }
+    subject(:last_response) { patch("/articles/#{article.external_id}", json) }
+
+    before do
+      allow_any_instance_of(UserPolicy::Scope).to receive(:resolve)
+        .and_return(user_policy_scope)
```

Comment:
This spec would actually pass even if we didn't handle `UserPolicy::Scope` at all, so I guess these `user_policy_scope` things could be removed.